config/default/capm3/namespace.yaml

```diff
@@ -2,3 +2,8 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: capm3-system
+  labels:
+    kubernetes.io/metadata.name: capm3-system
```

Comment:
Do we need this? Seems like duplication if the L4.

Reply:
I'll remove this, I don't think it is required.